tests/integration/cli/test_prune.py

```diff
@@ -0,0 +1,117 @@
+import os.path as osp
+from collections import Counter
+from unittest import TestCase
+
+import numpy as np
+
+from datumaro.cli.util.project import parse_dataset_pathspec
+from datumaro.components.annotation import Caption, Label
+from datumaro.components.dataset import Dataset
+from datumaro.components.dataset_base import DatasetItem
+from datumaro.components.media import Image
+from datumaro.util.scope import scope_add, scoped
+
+from ...requirements import Requirements, mark_requirement
+
+from tests.utils.test_utils import TestDir
+from tests.utils.test_utils import run_datum as run
+
+
+class PruneTest(TestCase):
```

Comment:
Use `pytest` for the newly created tests

Reply:
Fix in [4258c20](https://github.com/openvinotoolkit/datumaro/pull/1058/commits/4258c2060f46c723b1ecd46ce603b601bee77ade).